lib/hocon/impl/simple_config_list.rb

```diff
@@ -21,6 +24,8 @@ def initialize(origin, value, status = ResolveStatus.from_values(value))
     end
   end
 
+  def_delegators :@value, :[], :include?, :empty?, :size, :index, :rindex, :each, :map
+
```

Comment:
I see ...

@cprice404 @fpringvaldsen - what do y'all think about this stuff?  I'm okay with it, I suppose, although I can see an argument for writing out the methods the long way - again, to increase consistency with the java version.